packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -18,6 +18,28 @@ import sendPromptToJoinOrg from '../../../utils/sendPromptToJoinOrg'
 import {makeDefaultTeamName} from 'parabol-client/utils/makeDefaultTeamName'
 import isCompanyDomain from '../../../utils/isCompanyDomain'
 import {DataLoaderWorker} from '../../graphql'
+import getTeamsByIds from '../../../postgres/queries/getTeamsByIds'
+import acceptTeamInvitation from '../../../safeMutations/acceptTeamInvitation'
+import isValid from '../../isValid'
+import {Team} from '../../../postgres/queries/getTeamsByIds'
+
+const getAutoJoinTeams = async (
+  usersWithDomainTeams: Team[] | never[],
```

Comment:
-1 There is no reason to have this `| never[]` type here, it could just be a more conventional `| null`.